docs/coding-conventions.md

```diff
@@ -0,0 +1,89 @@
+# Coding conventions
+
+This document outlines a collection of guidelines, style suggestions, and tips
+for writing code in the KubeVirt project. It is partially based on [Kubernetes
+Coding Conventions](https://github.com/kubernetes/community/blob/master/contributors/guide/coding-conventions.md).
+
+- [Go Code Review Comments](https://github.com/golang/go/wiki/CodeReviewComments)
```

Comment:
The link redirects you to https://go.dev/wiki/CodeReviewComments

Reply:
Thanks, will fix it